### PR TITLE
Enable Travis for all Tests

### PR DESCRIPTION
**Pull request by shinobu**
This pullrequest enables the integration tests for Erfurt.

### CODE REVIEW

File: library/Erfurt/config.ini-dist-mysql [hunk 1]
```diff
@@ -7,8 +7,8 @@
 store.backend = zenddb ; zenddb, virtuoso
 
 store.zenddb.dbname   = ontowiki
-store.zenddb.username = ow
-store.zenddb.password = ow
+store.zenddb.username = travis
+store.zenddb.password = ''
```
Comment (white-gecko) [inline library/Erfurt/config.ini-dist-mysql]: Please revert these changes

File: .travis.yml [hunk 2]
```diff
@@ -1,25 +1,55 @@
...
+  - sudo apt-get -qq update
+  - sudo apt-get install -y raptor2-utils
   - phpenv config-rm xdebug.ini
+  - mysql -e 'CREATE DATABASE erfurt_TEST;'
 
 install:
   - travis_retry composer install --no-interaction
 
 before_script:
+  - cp ./tests/config.ini.dist.travis ./tests/config.ini.dist
```
Comment (white-gecko) [inline .travis.yml]: The file should be copied to `./tests/config.ini`

File: .travis.yml [hunk 3]
```diff
@@ -1,25 +1,55 @@
...
 
 php:
-  - 5.4
-  - 5.6
-  - 7.0
-  - 7.1
+  - '5.4'
+  - '5.6'
+  - '7.0'
+  - '7.1'
```
Comment (white-gecko) [inline .travis.yml]: Please don't do such changes.

File: tests/scripts/travis/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+- `install-services.sh` to handle the install of additional services
+
+## SPARQL services
+
+This file and scripts are taken form the SemanticMediaWiki repo and only selected files where copied for running tests with virtuoso.
+It is taken as of commit 721a63d3a73400300f73e0a088196a6ed0fe5afd from https://github.com/SemanticMediaWiki/SemanticMediaWiki.
+The unsupported services are marked with *Endabled* "No".
```
Comment (white-gecko) [inline tests/scripts/travis/README.md]: It would be good if you can also fix this typo

File: tests/scripts/travis/README.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+<table>
+	<tr>
+		<th>Endabled</th>
```
Comment (white-gecko) [inline tests/scripts/travis/README.md]: same here

File: tests/scripts/travis/install-extensions.sh [hunk 6]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```
Comment (white-gecko) [inline tests/scripts/travis/install-extensions.sh:1]: Please copy these files as they are in one commit and apply your changes in a second commit

**Review by white-gecko — CHANGES_REQUESTED** [review]

**Comment by shinobu** [comment]
those changes should fix the issues @white-gecko @pfrischmuth

**Comment by white-gecko** [comment]
@shinobu I've added another commit to get the travis file consistent. Please squash this into on of your commits. So we have two commits in the end.

**Comment by white-gecko** [comment]
The final changes (ignoring whitespace) are here: https://github.com/AKSW/Erfurt/pull/141/files?w=1

**Comment by shinobu** [comment]
done @white-gecko

**Review by white-gecko — APPROVED** [review]

**Pull request merged**